### DISCUSSION TITLE
Installation on Hetzner, Self Hosted Kubernetes

### DISCUSSION THREAD

**Discussion by clickersmudge** [keycloak/keycloak#15262] [Q&A]
Hi,
I try to using the Keycloak Operator Installation.
Cluster on Hetzner Cloud.
Step one (no problem)
https://www.keycloak.org/operator/installation
Step two (no problem)
https://www.keycloak.org/operator/basic-deployment

After doing the entire tutorial. I cannot connect to the Admin Console.
```
$ kubectl get pods -A
default                example-kc-0                                                      0/1     CrashLoopBackOff   17 (110s ago)   73m
```

```
$ kubectl describe pod -n default example-kc-0
Events:
  Type     Reason     Age                   From     Message
  ----     ------     ----                  ----     -------
  Warning  Unhealthy  59m (x43 over 73m)    kubelet  Liveness probe failed:
  Normal   Pulling    14m (x16 over 74m)    kubelet  Pulling image "quay.io/keycloak/keycloak:20.0.0"
  Warning  BackOff    4m7s (x299 over 72m)  kubelet  Back-off restarting failed container
```

```
$ kubectl get services -A
default                example-kc-discovery        ClusterIP      None            <none>                                                7800/TCP         74m
default                example-kc-service          ClusterIP      10.240.25.193   <none>                                                8443/TCP         74m
default                keycloak-operator           ClusterIP      10.240.21.202   <none>                                                80/TCP           86m
```

```
$ kubectl get ingress
example-kc-ingress   <none>   keycloak.mydomain.com             80      75m
```

**Comment by codingmanny** [comment]
@clickersmudge did it run in the end?

**Reply by codespearhead** [reply]
This discussion was created over a year ago. If you're encountering a similar problem, could you provide us with the console output from your browser as well as the Keycloak server logs when you attempt to access the Admin Console.

**Reply by codingbjoern** [reply]
i haven't started yet. I'm in design phase. not sure at the moment, if i really want to put keycloak in kubernetes, or in plesk

**Reply by codespearhead** [reply]
Plesk is closed-source, so you should direct that question to the vendor or their official community forum if it exists.

As for Kubernetes, please refer to the Kubernetes Guide in the "Getting Started" section of the documentation [1].

[1] https://www.keycloak.org/guides
